packages/lib/src/async-table/async-table.wc.svelte

```diff
@@ -489,11 +492,23 @@
     exportDataGeneric(option);
     isOpen = false;
   };
+
+  function handleReset() {
+    headers = defaultHeaders;
+    headers.forEach((header) => {
+      header.disabled = false;
+    });
+  }
```

Comment:
This actually isn't correct. The default state of a table might be to have some columns disabled initially. 

https://github.com/Jaspero/web-components/blob/f4dd2f9c5378ae9b135a607535975fc6a7709c17/packages/lib/src/async-table/async-table.wc.svelte#L469-L483

This is where we pull column order from the database and override the default. You need to save a reference of the default and in set it as the current headers in `handleReset()`